src/brogue/Items.c

```diff
@@ -5973,6 +5973,17 @@ void throwItem(item *theItem, creature *thrower, pos targetLoc, short maxDistanc
                     deleteItem(theItem);
                     return;
                 }
+
+                if ((theItem->category & POTION)
+                    && monst->creatureState == MONSTER_ALLY
+                    && theItem->kind == POTION_LIFE) {
+                        sprintf(buf, "your %s ally catches the potion and guzzles down the whole thing.", monst->info.monsterName);
```

Comment:
I'm not sure `guzzles` is very brogue-feeling wording, brogue being written in a rather serious way. `your %s ally catches the potion and gulps the whole thing down` or `your %s ally catches the potion and drinks it completely`. For discussion.

Reply:
I wonder if we just didn't have a message for this. There are some weird rare edge cases (eg wisp, kraken) where the anthro catching and drinking description might not make sense

Reply:
`empowerMonster` has `sprintf(buf, "%s looks stronger", theMonsterName);`

Perhaps "looks healthier" ?

Reply:
Nevermind. I see there is a message later for healing and health increase. I agree with removing the catch/drink message.